.github/workflows/validate-install-from-source.yml

```diff
@@ -0,0 +1,28 @@
+name: validate-install-from-source
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  docker:
+    name: ${{matrix.vector.image}}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        vector:
+        - image: ubuntu
+        - image: debian
+        - image: linuxmintd/mint20-amd64
+        - image: fedora
+        - image: centos
+        - image: redhat/ubi8
+        - image: alpine
+    container: ${{matrix.vector.image}}
+    steps:
+      - uses: actions/checkout@v1
+      # Run install from source script, passing an empty string to
+      # `--sudo-cmd` since containers run as root.
```

Comment:
I'm wondering if we can avoid this `--sudo-cmd=""` here by updating the script to do something like:

```shell
if [ -z "$SUDO_CMD" &&  ]; then
    if [ `id -u` != 0 ]; then
        SUDO_CMD=sudo
    fi
fi
```

If we're already effectively `root`, there's no need for `sudo`.

Reply:
Lovely! Thank you.